src/mappings/v2/setproperty.ts

```diff
@@ -0,0 +1,32 @@
+import { Optional } from '@kodadot1/metasquid/types'
+import { SetProperty } from '@kodadot1/minimark/v2'
+
+import { unwrap } from '../utils'
+import logger, { logError } from '../utils/logger'
+import { Action, Context } from '../utils/types'
+import { getSetProperty } from './getters'
+
+const OPERATION = Action.SETPROPERTY
+
+export async function setProperty(context: Context) {
+  let interaction: Optional<SetProperty> = null
+
+  try {
+    const getter = getSetProperty
+    const { value: interaction, caller, timestamp, blockNumber, version } = unwrap(context, getter);    
+
+    // logger.success(`[${OPERATION}] NEW PRIORITY ${interaction.value} for ${nft.id} from ${caller}`)
+
+    // TODO: add logic for accepting resource
```

Comment:
Will this be done in this PR?

Reply:
99.99% not :)
- need new property entity